python/ray/llm/_internal/serve/deployments/routers/router.py

```diff
@@ -77,6 +77,31 @@
 logger = get_logger(__name__)
 
 T = TypeVar("T")
+
+
+def _sanitize_chat_completion_request(
```

Comment:
should this be only applied to completion response? What about the other returned types?